tests/TransactionTest.php

```diff
@@ -0,0 +1,226 @@
+<?php
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\DB;
+
+class TransactionTest extends TestCase
+{
+    protected $insertData = ['name' => 'klinson', 'age' => 20, 'title' => 'admin'];
+    protected $originData = ['name' => 'users', 'age' => 20, 'title' => 'user'];
+    protected $connection = 'mongodb_repl';
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        User::on($this->connection)->truncate();
+        User::on($this->connection)->create($this->originData);
+    }
+
+    protected function getEnvironmentSetUp($app)
+    {
+        if (version_compare(env('MONGO_VERSION'), '4', '<')) {
```

Comment:
Same as with the other test, we shouldn't rely on an env variable for the MongoDB version, but we may be able to drop this check entirely.

Reply:
Done